internal/services/datafactory/data_factory_linked_service_azure_file_storage_resource.go

```diff
@@ -251,7 +243,7 @@ func resourceDataFactoryLinkedServiceAzureFileStorageRead(d *pluginsdk.ResourceD
 			return nil
 		}
 
-		return fmt.Errorf("retrieving Data Factory Linked Service Azure File Storage %q (Data Factory %q / Resource Group %q): %+v", id.Name, id.FactoryName, id.ResourceGroup, err)
+		return fmt.Errorf("retrieving Data Factory Azure File Storage %s: %+v", *id, err)
```

Comment:
The `*id` always contains the resource name when printing:
`Linked Service: (Name "acctest..." / Factory Name "acctest..." / Resource Group "acctest...")`
I decided to remove it since it becomes superfluous.